plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2961,6 +2961,7 @@ dialog.tools.inject_tags.tag.blocks.minecraft.wooden_fences=Used in 1.16.x to de
 dialog.tools.inject_tags.tag.blocks.minecraft.walls=Used by pathfinding and to determine connections with other walls, panes, bars, and gates.
 dialog.tools.inject_tags.tag.blocks.minecraft.small_flowers=Bees can pollinate these blocks, endermen can hold these blocks
 dialog.tools.inject_tags.tag.blocks.minecraft.tall_flowers=Bees can pollinate these blocks
+dialog.tools.inject_tags.tag.blocks.minecraft.bee_growables=Bees can pollinate these blocks
```

Comment:
It is a bit strange tall_flowers and bee_growables have the same description. There is probably some difference between these two?

Reply:
The `tall_flowers` tag isn't really useful as it is used to populate the `flowers` tag, which is then used to indicate if bees can pollinate them.

Reply:
Nevermind, there is a difference, and this PR isn't about `tall_flowers` so who cares

Reply:
> so who cares

the users that would see two tags with the same description

Reply:
don't mark conversations resolved by yourself, please

Reply:
Ok well it's outdated I changed the description, you can check the new one